apps/mudita-center-e2e/src/page-objects/home.page.ts

```diff
@@ -0,0 +1,11 @@
+import Page from "./page";
+
+class HomePage extends Page {
+    get homeHeader() {
+        return $("h2*=Welcome to Mudita Center")
+    }
+
+    get notNowButton() {
+        return $("button*=Not now")
+    }
+} export default new HomePage()
```

Comment:
- Prettier does not work in this file.
- Missing license.
Let's meet to discuss how we can automate this.

Reply:
corrected